### ISSUE TITLE
ChatComposite messages box won't resize its width according to the parent div

### ISSUE THREAD

**Issue by AlbertoPerez8** [Azure/communication-ui-library#2993]
The pre-built ChatComposite component is failing to resize the width of the container that displays the messages in relation to the parent div that is holding it. I find it weird since the Topic segment of the composite is able to resize with no issues, but the messages box can't. 

![image](https://user-images.githubusercontent.com/36648131/234881696-ea9de615-a843-43de-93bd-17b96c8925a2.png)

![image](https://user-images.githubusercontent.com/36648131/234881762-f19ea710-ddcd-4ac4-8578-084f066f44e1.png)

When inspecting the composite closer with Chrome's developer tools, I noticed that the specific div that won't adjust has a  <div class="ms-Stack css-150"> tag, when changing the 150 for the 147 that the parent div had in it's class I saw that it would then adjust its width.

I was running the composite inside a component held in a single div tag with width and height values of 100vw.

![image](https://user-images.githubusercontent.com/36648131/234883369-906d1b9e-c8d9-448b-854c-63bdd82337ff.png)

And the composite running inside the component is virtually exactly the same as the sample code provided at https://azure.github.io/communication-ui-library/?path=/docs/composites-chat-basicexample--basic-example, with the only changes made being the changing of the component class name and the removal of the participantsPane props from the ChatComposite itself.

- `@azure/communication-react` npm package version [^1.5.0]
- `@azure/communication-calling` npm package version [^1.4.4]
- `@azure/communication-chat` npm package version [^1.2.0]

- OS & Device: Windows on PC
- Browser: Chrome 
- Browser Version: 112.0.5615.138 (Official Build) (64-bit)

**Comment by palatter** [comment]
Thank you for reporting this issue. We will investigate and update this thread.

**Comment by AlbertoPerez8** [comment]
I did a barebones React project where I reused the same code for the component holding the ChatComposite, and while it also did not auto adjust width, I noticed that it did not have the user avatars clipping into the left side of the chat messages. 
![image](https://user-images.githubusercontent.com/36648131/235172942-db2b46eb-f6d5-4bb1-86ab-c36e5771a0fc.png)
![image](https://user-images.githubusercontent.com/36648131/235174134-2ccaaf41-8887-47b1-8b94-6489a629a9c0.png)
![image](https://user-images.githubusercontent.com/36648131/235174568-a1566156-293c-44e5-874c-0b6ef0cad836.png)
![image](https://user-images.githubusercontent.com/36648131/235174636-5bb972c7-5784-425e-a0a8-bc811468b823.png)

The packages are the exact same ones between projects.
![image](https://user-images.githubusercontent.com/36648131/235173220-8c85f05c-1757-4f43-81d2-12b8e71b5c25.png)

Does the ChatComposite have incompatibility issues with Teams Toolkit applications. When I used the videoComposite I didn't run into these types of issues.

**Comment by palatter** [comment]
After investigating I found that this behavior is expected, as we have defined the width to have this constant value. I have proposed that we could add support for customizing this or disabling it in the future. 

As for the avatars clipping, you are only seeing this when in a teams toolkit application?

**Comment by AlbertoPerez8** [comment]
Yes, these behaviors only appear when being run from a teams toolkit application, and not from a standard react app.

**Comment by emlynmac** [comment]
Can you try hosting in an iframe? We've had success in this case during investigations.

**Comment by AlbertoPerez8** [comment]
Is this way how you would implement it with an iframe?
![image](https://github.com/Azure/communication-ui-library/assets/36648131/4ca09280-6cea-4187-9e69-f0fb030309d4)

Also, while messing around with the CSS using chrome's dev tools, we found that altering the tag by inserting 
![image](https://github.com/Azure/communication-ui-library/assets/36648131/fe2a585b-18ed-4615-b8e4-a9899d5ac627)
helped with the presentation. Could this be done in the react project by adding a css folder somewhere in the App.tsx or Index.tsx files? And if so, would it be safe to do so, is that property isolated and only used to display the messages?

**Comment by palatter** [comment]
I did not observe this issue by creating a stand-alone react app that housed the ChatComponent. I then used an iframe in the teams toolkit app to point to the site that was serving the stand-alone chat component react app. 
 
Would this workflow be viable for you?

**Comment by palatter** [comment]
![image](https://github.com/Azure/communication-ui-library/assets/73612854/4baa7fac-78d8-4f76-af4e-9f5de137b437)

**Comment by AlbertoPerez8** [comment]
So I was able to get a work around done that fixes both the clipping issue and the original resizing chat complaints by forcing some CSS stylings in the App.css files. Using Chrome webtools I looked for the individual components and was able to provide some sizing properties and get the stylings I wanted. 
![image](https://github.com/Azure/communication-ui-library/assets/36648131/fee18464-f4f3-42b9-a01c-09dbb6f74dec)
![image](https://github.com/Azure/communication-ui-library/assets/36648131/900cba12-a29f-4c06-a213-f2b9527ee907)
However, since it's a CSS workaround, I am not sure that this solution can be of help to others through the exact means and it would be nice if the chat width adjustment can be a feature to be implemented in the future, or some documentation on the styling structure of the composites. Either way, thanks for the help and solution recommendations.
